controllers/routes.js

```diff
@@ -26,7 +25,6 @@ const { basicContent, flexibleContent, staticPage } = require('./common');
 const toplevel = {
     path: '',
     showInNavigation: true,
-    langTitlePath: 'global.nav.home',
```

Comment:
We changed how the navigation is constructed with the new header and this isn't needed anymore.